### DISCUSSION TITLE
Possible to prevent a server from being used as a build server?

### DISCUSSION THREAD

**Discussion by andrewkouri** [coollabsio/coolify#4709] [Improvement Requests]
I would like to _never_ use my production servers as build servers. Occasionally they will get selected for builds, and slow down the database/app.  I would like to set this to 0, (i.e. force reliance on a dedicated build server) but it will not let me.

<img width="680" alt="image" src="https://github.com/user-attachments/assets/82f5199a-d8b9-4b1e-9fc1-f44490e7601f" />

**Comment by schoenwaldnils** [comment]
Same here. I run coolify on a weak vm and let the deployments get build on another machine. I'd love to disable building on the production server. Just queue the deployment and wait for the build server to be free
